app/subscriber/src/components/tool-bar/AddToReportMenu.tsx

```diff
@@ -0,0 +1,131 @@
+import { TooltipMenu } from 'components/tooltip-menu';
+import React from 'react';
+import { FaFileExport, FaPlay } from 'react-icons/fa6';
+import { toast } from 'react-toastify';
+import { useReports } from 'store/hooks';
+import { useProfileStore } from 'store/slices';
+import { IContentModel, IReportModel, Row } from 'tno-core';
+
+import * as styled from './styled';
+import { newReportInstance, toInstanceContent } from './utils';
+
+export interface IAddToReportMenuProps {
+  content: IContentModel[];
+}
+export const AddToReportMenu: React.FC<IAddToReportMenuProps> = ({ content }) => {
+  const [{ updateReport, findMyReports, getReport }] = useReports();
+  const [{ myReports }] = useProfileStore();
+  const [activeReport, setActiveReport] = React.useState<IReportModel | null>(null);
+  const [reportId, setReportId] = React.useState<number | null>(null);
+  React.useEffect(() => {
+    if (!myReports.length) {
+      findMyReports().catch(() => {});
+    }
+    // Only do this on init.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  /** Adds the content to the active report. */
+  const addContentToReport = React.useCallback(
+    (section: string) => {
+      if (!activeReport) {
+        return;
+      }
+
+      const convertedContent = toInstanceContent(content, activeReport, section);
+      const update = (report: IReportModel) => {
+        updateReport(report, true)
+          .then(() => toast.success(`${content.length} storie(s) have been added to report.`))
+          .catch(() => {});
+      };
+
+      if (!!activeReport.instances.length) {
+        // get the latest instance content and append to it
+        const instContent = [
```

Comment:
The current instance will always be `0` index position.  We return them in descending order always

Reply:
We never want to add content to a sent report instance.  A new instance will need to be generated in this case.
`ReportInstance.SentOn == null` to allow updating content.
We should add an error in the API if anyone attempts to update a sent instance.

Reply:
Gotcha, ok will update and based on below comments as well!